docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,132 @@
+- **Ephemeral Data Context:** Exists in memory, and does not persist beyond the current Python session.
+- **File Data Context:** Exists as a folder and configuration files. Its contents persist between Python sessions.
+- **Cloud Data Context:** Supports persistence between Python sessions, but additionally serves as the entry point for Great Expectations Cloud.
+
+- [Data Context tutorials](/docs/guides/setup/configure_data_contexts_lp)
+
+### The GX API
+
+A Data Context object in Python provides methods for configuring and interacting with GX.  These methods and the objects and additional methods accessed through them compose the GX public API.
+
+- [The GX API reference](/docs/reference/api_reference)
+
+### Stores
+
+Stores contain the metadata GX uses.  This includes configurations for GX objects, information that is recorded when GX validates data, and credentials used for accessing data sources or remote environments.  GX utilizes one Store for each type of metadata, and the Data Context contains the settings that tell GX where that Store should reside and how to access it.
+
+- [Stores tutorials](/docs/guides/setup/setup_overview_lp)
+
+### Data Docs
+
+Data Docs are human-readable documentation generated by GX.  Data Docs describe the standards that you expect your data to conform to, and the results of validating your data against those standards.  The Data Context manages the storage and retrieval of this information.
+
+You can configure where your Data Docs are hosted.  Unlike Stores, you can define configurations for multiple Data Docs sites.  You can also specify what information each Data Doc site provides, allowing you to format and provide different Data Docs for different use cases.
+
+- [Data Docs tutorial](/docs/guides/setup/configuring_data_docs/host_and_share_data_docs)
+
+## Datasources
+
+![Datasource overview](./images_overview/datasource_flowchart.png)
+
+Datasources connect GX to source data such as CSV files in a folder, a PostgreSQL database hosted on AWS, or any combination of data formats and environments. Regardless of the format of your source data or where it resides, Datasources provide GX with a unified API for working with it.
+
+- [Datasource tutorials](/docs/guides/connecting_to_your_data/connect_to_data_lp)
+
+### Data Assets and Batches
+
+Data Assets are collections of records within a Datasource.  While a Datasource tells GX how to connect to your source data, Data Assets tell GX how to organize that data. Although the records in your Data Assets can correspond directly to the contents of tables or files in your source data they do not necessarily need to. For instance, you could combine multiple tables worth of records in a SQL Datasource into a single Query Data Asset that joins the tables in question.  For a File Datasource, you could use regular expressions to define a Data Asset as the contents of all of the `.csv` files in a specific subfolder.
```

Comment:
I think it would be helpful to have a more concrete example cited here and perhaps why you may want to do it.